powerauth-test-server/src/main/java/com/wultra/security/powerauth/app/testserver/model/request/OperationApproveInternalRequest.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * PowerAuth test and related software components
+ * Copyright (C) 2022 Wultra s.r.o.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.wultra.security.powerauth.app.testserver.model.request;
+
+import lombok.Data;
+
+/**
+ * Request for approving an operation.
+ *
+ * @author Petr Dvorak, petr@wultra.com
+ */
+@Data
+public class OperationApproveInternalRequest {
```

Comment:
The class looks dense, can we improve the formatting a bit?

Reply:
No, it's OK.